cocos/core/math/quat.ts

```diff
@@ -82,6 +87,7 @@ export class Quat extends ValueType {
     }
 
     /**
+     * @en Set quaternion rotation is the shortest path between two vectors, the default two vectors are normalized
```

Comment:
Sets the out quaternion with the shortest path orientation between two vectors, considering both vectors normalized